docs/how_to/env_var.md

```diff
@@ -41,47 +56,59 @@ Typically, you wouldn't need to change these settings, but they are listed here
 
 ## Execution Options
 
-* MXNET_EXEC_BULK_EXEC_INFERENCE (default=1)
+* MXNET_EXEC_BULK_EXEC_INFERENCE
+  - Values: 0(false) or 1(true) ```(default=1)```
   - If set to `1`, during inference MXNet executes the entire computation graph in bulk mode, which reduces kernel launch gaps in between symbolic operators.
-* MXNET_EXEC_BULK_EXEC_TRAIN (default=1)
+* MXNET_EXEC_BULK_EXEC_TRAIN
+  - Values: 0(false) or 1(true) ```(default=1)```
   - If set to `1`, during training MXNet executes the computation graph as several subgraphs in bulk mode.
-* MXNET_EXEC_BULK_EXEC_MAX_NODE_TRAIN (default=15)
+* MXNET_EXEC_BULK_EXEC_MAX_NODE_TRAIN
+  - Values: Int ```(default=15)```
   - The maximum number of nodes in the subgraph executed in bulk during training(not inference). Setting this to a larger number may reduce the degree of parallelism for multi-GPU training.
 
 ## Control the Data Communication
 
-* MXNET_KVSTORE_REDUCTION_NTHREADS (default=4)
+* MXNET_KVSTORE_REDUCTION_NTHREADS
+  - Values: Int ```(default=4)```
 	- The number of CPU threads used for summing big arrays.
-* MXNET_KVSTORE_BIGARRAY_BOUND (default=1e6)
+* MXNET_KVSTORE_BIGARRAY_BOUND
+  - Values: Int ```(default=1e6)```
 	- The minimum size of a "big array."
 	- When the array size is bigger than this threshold, MXNET_KVSTORE_REDUCTION_NTHREADS threads are used for reduction.
-* MXNET_ENABLE_GPU_P2P (default=1)
-    - If true, MXNet tries to use GPU peer-to-peer communication, if available,
-      when kvstore's type is `device`
+  - This parameter is also used as a load balancer in kvstore. It controls when to partition a layer's weights to all the servers. If weights size of each layer is less than MXNET_KVSTORE_BIGARRAY_BOUND then, weights are sent to a single randomly picked server otherwise it is partitioned to all the servers.
```

Comment:
a layer's weights -> a single weight

Decision is based on individual weights, not layers.